problem_builder/mentoring.py

```diff
@@ -905,6 +905,31 @@ def workbench_scenarios():
         """
         return loader.load_scenarios_from_path('templates/xml')
 
+    def student_view_data(self):
+        """
+        Returns a JSON representation of the student_view of this XBlock,
+        retrievable from the Course Block API.
+        """
+        components = []
+        for child_id in self.children:
+            block = self.runtime.get_block(child_id)
+            if hasattr(block, 'student_view_data'):
+                components.append(block.student_view_data())
+        return {
+            'max_attempts': self.max_attempts,
+            'extended_feedback': self.extended_feedback,
+            'feedback_label': self.feedback_label,
+            'components': components,
+            'messages': {
+                message_type: get_message_label(message_type)
```

Comment:
We don't want the message Studio label here (`get_message_label`) - we don't care about that. We want the result of [`self. get_message_content(message_type)`](https://github.com/open-craft/problem-builder/blob/bfa131b63928ef2e60a1c9d50fd22ced957cb25c/problem_builder/mixins.py#L102) which will either be `None` or an HTML string.